installing/installing_with_agent_based_installer/preparing-an-agent-based-installed-cluster-for-mce.adoc

```diff
@@ -0,0 +1,30 @@
+:_content-type: ASSEMBLY
+[id="preparing-an-agent-based-installed-cluster-for-mce"]
+= Preparing an Agent-based installed cluster for multicluster engine for Kubernetes (MCE) Operator
+include::_attributes/common-attributes.adoc[]
+:context: preparing-an-agent-based-installed-cluster-for-mce
+
+toc::[]
+
+You can install the multicluster engine for Kubernetes(MCE) Operator and deploy a hub cluster with the Agent-based Installer.
+The following procedure is partially automated and requires manual steps after the initial cluster is deployed.
+
+== Prerequisites
+* You have read the documentation on link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html/multicluster_engine/multicluster_engine_overview[multicluster engine for Kubernetes].
+* You have read the documentation on xref:../../storage/persistent_storage/persistent-storage-local.adoc#persistent-storage-using-local-volume[persistent storage using local volumes].
+* You have read the documentation on xref:../../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.adoc#ztp-deploying-far-edge-clusters-at-scale[using ZTP to provision clusters at the network far edge].
+* You have read the documentation on xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#preparing-to-install-with-agent-based-installer[preparing to install with the Agent-based Installer].
+* You have read the documentaion xref:../../installing/disconnected_install/index.adoc#installing-mirroring-disconnected-about[about disconnected installation mirroring].
+* You have access to the internet to obtain the necessary container images.
+* You have installed the OpenShift CLI (oc).
+* You have a configured local mirror registry for disconnected installation mirroring.
```

Comment:
```suggestion
* If you are installing in a disconnected environment, you have a configured local mirror registry for disconnected installation mirroring.
```
Rewording suggestion so that users don't think this is a requirement for connected installs